loklak.php

```diff
@@ -149,4 +149,41 @@ public function settings() {
 			return json_encode($request, true);
 		}
 	}
+
+	public function suggest($query, $count=null, $order=null, $orderby=null, $since=null, $until=null) {
```

Comment:
The `query` by itself can be `""` empty which means the request for `loklak.org/api/suggest.json` also returns a response. This should also be considered.